credentials/FormXApi.credentials.ts

```diff
@@ -6,6 +6,7 @@ import {
 	IHttpRequestOptions,
 	INodeProperties,
 } from 'n8n-workflow';
+// @ts-ignore
```

Comment:
Does adding `make setup` in CI fix the pipeline? I'd prefer we don't hack around type checking to fix CI issues.

Reply:
Should do. Let me test